qemux86-64.robot

```diff
@@ -31,5 +33,8 @@ Git pushing to application
   Push ${application_repo} to application ${application_name}
 Check if device is running the pushed application (Tries for 300 s)
   Wait Until Keyword Succeeds    30x    10s    Device ${device_uuid} log should contain Hello
+Check if test environment variable is present
+  Adding ENV variable ${variable_name} with value ${variable_value} to application ${application_name}
```

Comment:
We can make the variable name static - say Hello with value world? No point in having it as a global suite variable if we arent planning to use it again.